crates/shared/src/arguments.rs

```diff
@@ -135,6 +135,10 @@ pub struct Arguments {
     /// If quasimodo should use internal buffers to improve solution quality.
     #[structopt(long, env)]
     pub quasimodo_uses_internal_buffers: bool,
+
+    /// If mipsolver should use internal buffers to improve solution quality.
+    #[structopt(long, env)]
+    pub mipsolver_uses_internal_buffers: bool,
```

Comment:
uber-naming-nit:
For consistency, can we name them either:
```
quasimodo_solver_uses_internal_buffers
mip_solver_uses_internal_buffers
```

Or (my personal preference):
```
quasimodo_uses_internal_buffers
mip_uses_internal_buffers
```

Reply:
Thanks, makes sense!